pkg/agent/config.go

```diff
@@ -44,51 +45,45 @@ type MQTTConf struct {
 	PrivKeyPath string          `json:"priv_key_path" toml:"priv_key_path" mapstructure:"priv_key_path"`
 	CA          []byte          `json:"-" toml:"-"`
 	Cert        tls.Certificate `json:"-" toml:"-"`
+	ClientCert  string          `json:"client_cert" toml:"client_cert"`
+	ClientKey   string          `json:"client_key" toml:"client_key"`
+	CaCert      string          `json:"ca_cert" toml:"ca_cert"`
 }
 
-type Heartbeat struct {
+type HeartbeatConf struct {
```

Comment:
Sometimes you call this `Conf` and sometime `Config`. Please pick one and stay consistent.

Reply:
I think `Config` would be better, looking at `NewConfig()` name...